src/components/Header/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Navbar, Container, Nav } from "react-bootstrap";
+
+const Header = () => {
```

Comment:
Remove the bootstrap code if you're not planning on using it

Reply:
Ya I'll go back in and clean up all the Bootstrap artifacts.. sorry!